wrappers/services.go

```diff
@@ -137,10 +137,15 @@ func stopService(sysd systemd.Systemd, app *snap.AppInfo, inter interacter) erro
 	return nil
 }
 
+// StartServicesFlags carries extra flags for StartServices.
+type StartServicesFlags struct {
+	Enable bool
+}
+
 // StartServices starts service units for the applications from the snap which
 // are services. Service units will be started in the order provided by the
 // caller.
-func StartServices(apps []*snap.AppInfo, inter interacter, tm timings.Measurer) (err error) {
+func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServicesFlags, inter interacter, tm timings.Measurer) (err error) {
```

Comment:
Nitpick, `disabledSvcs` is unused in this branch. I understand it comes in later.

Reply:
Yes, this is to slightly reduce the diff before the main PR.